react/react1/week1/todolist/src/components/AddTodo.js

```diff
@@ -0,0 +1,5 @@
+function AddTodo({ addTodo }) {
```

Comment:
you don't need this button in any other part of the app, so don't create a component only for this
it takes memory and reduces the app performance when you create unnecessary components, this button could be on the todo list component.